packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -5337,14 +5449,64 @@ addEventListener('fetch', event => {});`
         Object {
           "debug": "",
           "err": "",
-          "out": "--dry-run: exiting now.",
+          "out": "Total Upload: 5.00 KiB / gzip: 2.00 KiB
+        --dry-run: exiting now.",
           "warn": "[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mEnabling node.js compatibility mode for built-ins and globals. This is experimental and has serious tradeoffs. Please see https://github.com/ionic-team/rollup-plugin-node-polyfills/ for more details.[0m
 
         ",
         }
       `);
     });
   });
+
+  describe("bundle reporter", () => {
+    it("should output bundle size message", async () => {
+      fs.writeFileSync(
+        "./text.txt",
+        `${new Array(100000)
+          .fill("Try not. Do or do not. There is no try.")
+          .join("")}`
+      );
+
+      fs.writeFileSync(
+        "./hello.html",
+        `<!DOCTYPE html>
+      <html>
+        <body>
+            <h2>Hello World!</h2>
+        </body>
+      </html>
+      `
+      );
+      fs.writeFileSync(
+        "index.js",
+        `import hello from "./hello.html";
+    import text from "./text.txt";
+      export default {
+        async fetch(request) {
+          return new Response(json.stringify({ hello, text }));
+        },
+      };`
+      );
+      writeWranglerToml({
+        main: "index.js",
+      });
+      mockSubDomainRequest();
+      mockUploadWorkerRequest();
+      await runWrangler("publish");
+      expect(std).toMatchInlineSnapshot(`
+        Object {
+          "debug": "",
+          "err": "",
+          "out": "Total Upload: 3810.00 KiB / gzip: 12.00 KiB
```

Comment:
This is the test made to show the significant difference after compression with modules that have some content.

Reply:
The compression is this good because it's a repeated string which is amenable to compression, so we're only demonstrating the gzip algorithm. Let's reduce the size of the text file a bit to keep this test fast.

Reply:
I reduced the size of text being created for this test.